app/helpers.ts

```diff
@@ -0,0 +1,9 @@
+import { db } from "@/db/client";
+
+export async function getPolls() {
```

Comment:
You may want to call this getPublicPolls so it's clear what the goal of it is

Reply:
That's true. Thanks.